extensions/parquet/table/src/main/java/io/deephaven/parquet/table/location/ParquetTableLocation.java

```diff
@@ -598,67 +591,62 @@ public void pushdownFilter(
             columnIndices.add(resolvedColumn.columnIndex);
         }
 
+        final WritableRowSet mm;
```

Comment:
Tracking only maybe matches works for the currently implemented features. Adding other pushdown actions (like dictionary support) will involve tracking both match and maybe match.